README.md

```diff
@@ -19,5 +19,6 @@ For example care should be taken around handling:
 * nested JSON structures and other data type limitations
 
 1. [IBM Cloud Functions](./cloud-functions)
-2. [Apache Spark on Data Science Experience](./spark-on-dsx)
-2. [Node-RED on IBM Cloud](./node-red)
+1. [Apache Spark on Data Science Experience](./spark-on-dsx)
+1. [Node-RED on IBM Cloud](./node-red)
+1. [IBM Message Hub](./message-hub)
```

Comment:
From our discussion, let's point this to `./message-hub/README.md`.

Reply:
Fixed as part of the rebase after your merge of #6 (sorry for rebasing while under review)